Library/SQLite/SQLite3.red

```diff
@@ -0,0 +1,444 @@
+Red [
+	Title:   "Red SQLite3 binding"
+	Author:  "Oldes"
+	File: 	 %SQLite3.red
+	Rights:  "Copyright (C) 2017 David 'Oldes' Oliva. All rights reserved."
+	License: "BSD-3 - https:;//github.com/red/red/blob/master/BSD-3-License.txt"
+]
+#system [
+	#include %SQLite3.reds
+
+	#define TRACE(value) [
```

Comment:
It's a very bad idea to create R/S macro name that could easily collide with others code, or even the Red runtime library in the future.